Engine/Graphics/Direct3D/Render.cpp

```diff
@@ -2677,7 +2677,7 @@ void Render::_4A4CC9_AddSomeBillboard(stru6_stru1_indoor_sw_billboard *a1,
     float depth = 1000000.0;
     for (uint i = 0; i < (unsigned int)a1->uNumVertices; ++i) {
         if (a1->field_104[i].z < depth) {
-            depth = a1->field_104[i * 4].z;
+            depth = a1->field_104[i * sizeof(void *)].z;
```

Comment:
looks very strange
in any case using sizeof type as array index it's a really bad idea

Reply:
Sorry you are right, this is incorrect. Further inspecting the original code I believe this should be a1->field_104[i]